hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -363,6 +363,10 @@
       "ozone.scm.network.topology.schema.file";
   public static final String OZONE_SCM_NETWORK_TOPOLOGY_SCHEMA_FILE_DEFAULT =
       "network-topology-default.xml";
+  public static final String DFS_NETWORK_TOPOLOGY_AWARE_READ_ENABLED =
+      "dfs.network.topology.aware.read.enable";
+  public static final String DFS_NETWORK_TOPOLOGY_AWARE_READ_ENABLED_DEFAULT =
+      "true";
```

Comment:
should we set the default to false so that if the deployment does not config a topology we don't have the overhead.

Reply:
Currently, "dfs.network.topology.aware.read.enable" controls the client whether to read from the closest data node or not.  The location sort process always happens on the om/scm side. Do you suggest to make this property a server side configuration?

Reply:
Not at this time. Let's keep it as-is and revisit if we need to make this a server side option.